src/utils/vote-presets.js

```diff
@@ -200,8 +216,108 @@ const getVotePresets = ({ presetParams, setPresetParams, provider }) => {
       prepare: async () => {
         const params = Object.values(presetParams)
         if (params.length < 1) return null
+        const inflationData = prepareInfaltionData(params[0])
+
+        return prepareInflationProposal(
+          inflationData.Vault,
+          inflationData.EthPNT,
+          inflationData.ethPNTAddress,
+          settings.contracts.pNetworkAssociationGnosisSafeAddress,
+          inflationData.rawAmount
+        )
+      }
+    },
+    withdrawInflationAndPegin: {
+      id: 'withdrawInflationAndPegin',
+      name: 'Withdraw Inflation and Pegin',
+      description:
+        'Withdraw requested inflated ethPNT amount to the treasury and Pegin to an account on a different chain',
+      args: [
+        {
+          id: 'input-amount',
+          name: 'amount',
+          component: 'Input',
+          props: {
+            type: 'number',
+            style: {
+              fontSize: 15
+            },
+            placeholder: 'Amount ...',
+            value: presetParams[0] || '',
+            onChange: (_e) =>
+              setPresetParams({
+                ...presetParams,
+                0: _e.target.value
+              })
+          }
+        },
+        {
+          id: 'select-chain',
+          name: 'destinationChain',
+          component: 'ChainSelection',
+          props: {
+            onSelect: (_chainId) =>
+              setPresetParams({
+                ...presetParams,
+                1: _chainId
+              })
+          }
+        },
+        {
+          id: 'input-receiver-address',
+          name: 'receiverAddress',
+          component: 'Input',
+          props: {
+            style: {
+              fontSize: 15
+            },
+            placeholder: 'Receiver address ...',
+            value: presetParams[2] || '',
+            onChange: (_e) =>
+              setPresetParams({
+                ...presetParams,
+                2: _e.target.value
+              })
+          }
+        }
+      ],
+      prepare: async () => {
+        if (Object.values(presetParams).length < 2) return null
+        const amount = presetParams[0]
+        const destinationChainId = !presetParams[1] ? settings.chains[0].chainId : presetParams[1]
```

Comment:
I would put `bsc` as a default chain where to pegin to. And also I won't complain a comment saying what would be the default chain there beside this line
```suggestion
        const destinationChainId = !presetParams[1] ? settings.chains[0].chainId : presetParams[1] // defaults to arbitrum
```